### DISCUSSION TITLE
Is using mithril with cordova a good call for building native apps?

### DISCUSSION THREAD

**Discussion by EverettMcKay** [MithrilJS/mithril.js#2888] [General]
I started my js framework/library journey with React, but switched over to Mithril two years ago and have no desire to go back.

I want to restart a dormant React Native project and **am thinking that porting to mithril+cordova might be a good call. I see there is an encouraging discussion on this already, but it's 6 years old and the world has changed alot. Does anyone have any advice based on recent experience with using Mithril with native apps?** I will need Bluetooth integration, in case that makes a difference.

Btw: The reason the project is dormant is because I found React Native to be a super productivity killer and because of that, I ran out of runway. If I can finish the project in Mithril, I can make it work again. I'm easily 10x more productive in Mithril than RN.

**Comment by dead-claudia** [comment]
Cordova uses HTML for its UI, so I don't see any reason why Mithril _shouldn't_ work. And you wouldn't be the first to use Mithril + Cordova, either.

**Comment by boazblake** [comment]
Try ionic framework. You can use mithril to wire the web components together and use capacitor to create your native apps
